apps/zotonic_mod_zotonic_site_management/priv/skel/empty/src/SITE_sup.erl

```diff
@@ -0,0 +1,40 @@
+%% @author %%FULLNAME%%
+%% @copyright %%YEAR%% %%FULLNAME%%
+%% Generated on %%DATE%%
+%% @doc %%SITE%% root supervisor.
+
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+
+-module(%%SITE%%_sup).
+-author("%%FULLNAME%%").
+
+-behaviour(supervisor).
+
+%% API functions
```

Comment:
Could you please remove comments `%% API functions` and `%% Supervisor callbacks` here and in other `SITE_sup.erl`? Although they can be left. But since there isn't a lot of code here and it looks pretty self-explanatory, I think comments like this can be a bit redundant. But it's up to you to decide - it's just a recommendation :upside_down_face:.